### ISSUE TITLE
Teams Module issue

### ISSUE THREAD

**Issue by bnewman20** [freescout-helpdesk/freescout#2138]
On the teams module. 

Sometimes they will duplicate and won't be able to assign a ticket to the team. 

![image](https://user-images.githubusercontent.com/60909819/182929943-2e38f586-840c-4517-bd68-84129a536277.png)

Any thoughts?

**Comment by freescout-helpdesk** [comment]
Check errors in Manage -> Logs -> App Logs

**Comment by ilyakonovalenko** [comment]
Same issue here.

Logs shows:
```
Call to a member function isDeleted() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function isDeleted() on null at /var/www/XXX/data/www/XXX/Modules/Teams/Providers/TeamsServiceProvider.php:424)
```

**Comment by freescout-helpdesk** [comment]
> Call to a member function isDeleted() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function isDeleted() on null at /var/www/XXX/data/www/XXX/Modules/Teams/Providers/TeamsServiceProvider.php:424)

This error and duplicating folders have been fixed in Teams Module v1.0.3. But existing duplicated folders will not disappear -they can be deleted manually via DB  from `teams` table.
